incubator/insights-sumologic/README.md

```diff
@@ -0,0 +1,150 @@
+          namepass = [ "SqlListener*", "Commits", "Inserts", "Updates", "Deletes", "WriteThrottleTime", "Pending*", "Archive*Time", "Journal*Time" ]
+          fieldpass = [ "rate", "value" ]
+          data_format = "carbon2"
+        [[outputs.sumologic]]
+          url = "https://endpoint1.collection.eu.sumologic.com/receiver/v1/http/ZaVnC4dhaV0laeqZ1_FuM8b_P9gPRX4DIJzrJ8eaIy0Lz0rQH5o-bIARKiKf7M09suw6VFJpKSIt_3zv0hq80kcq0VivLeZWM3yXey7EXDG5YFFBaMWqMg=="
+          namepass = [ "Summary.*" ]
+          fieldpass = [ "raw" ]
+          data_format = "carbon2" 
+```
+
+### Sumo Logic Dashboards
+
+---
+
+1. [Sumo Logic Collectors](#sumo-logic-collectors) for NuoDB Insight data.
+
+2. [NuoDB Insights graphs](#insights-graphs-on-sumo-logic-dashboards) in Sumo Logic dashboards.
+
+3. [Kubernetes Graphs](#sumo-logic-kubernetes-graphs) in Sumo Logic (Optional)
+
+#### Sumo Logic Collectors
+
+---
+
+Website is https://www.sumologic.com
+
+Create a new account or use an existing account. For a new account, skip past the introductions and go straight to the dashboard. At the time of writing, the skip link was below the introduction text.
+
+Click on <b>Manage Data</b>: 
+
+![Manage Data](./images/sumologic0.png)
+
+Click on **Add Collector**
+
+![Add Collector](./images/sumologic1.png)
+
+Select **Hosted Collector**
+
+![Hosted Collector](./images/sumologic2.png)
+
+Add an appropriate name for the collector.
+
+Next is to add a source to the newly created collector. At the time of writing, the source was under "Generic" and is  "HTTP logs and metrics"
+
+![Generic_logs and metrics](./images/sumologic4.png)
+
+Add an appropriate name for the source for the collector. 
+
+Next is to copy and keep safe the Sumo Logic metrics endpoint. This will be needed in the section for the helm database values file setup. It will be something like:
+
+```
+https://endpoint1.collection.eu.sumologic.com/receiver/v1/http/ZaVnC4dhaV0laeqZ1_FuM8b_P9gPRX4DIJzrJ8eaIy0Lz0rQH5o-bIARKiKf7M09suw6VFJpKSIt_3zv0hq80kcq0VivLeZWM3yXey7EXDG5YFFBaMWqMg==```
+```
+
+![sumologic endpoint](./images/sumologic6.png)
+
+#### 
+
+#### Insights graphs on Sumo Logic dashboards
+
+---
+
+Click on Personal -> three dots to the right of the side bar and create a new folder.
+
+![Personal_Folder](./images/sumologic21.png)
+
+Click on the 3 dots to the right of the folder name and select Import
+
+![Import](./images/sumologic22.png)
+
+Upload the NuoDB dashboard JSON as a copy paste operation.  The JSON files that can be copy pasted can be found in the following locations
+
+```
+files/NuoDB_AdHoc.json
+files/NuoDB_DBA.json
+files/NuoDB_NOC.json
+```
+
+Click "Import" at the bottom of the pane.
+
+ ![Nuodb_Dashboard_JSON](./images/sumologic25.png)
+
+Set the Collector name and Source name to be the same as set in section 1.
```

Comment:
I don't see the collector and source names in any of the screenshots in section 1.

Reply:
The collector names and source names can be anything, just so long as its consistent. I'm not in favour of providing a name in just case users of the README think its mandatory names. 

Is there a better solution?

Reply:
My point is that it's not clear to the user where they can be taken from. It looks like both are provided by the user on some configuration screen in the Sumologic console. Maybe you just clarify this?

Reply:
I've added some clarification around the name of the Collector and Data Source.  Does that help?

> Add an appropriate name for the data source for the collector. This name can be anything, but will be used in the [Sumo Logic NuoDB graphs](https://github.com/nuodb/nuodb-helm-charts/tree/agibbons/insights-sumologic/incubator/insights-sumologic#insights-graphs-on-sumo-logic-dashboards) to select the data source.

> Tip: A Sumo Logic Collector may have many Data Sources. A Source Category value may be added to the Data Source meta data to help manage selection of data.

Reply:
Added a further link to refer back to the Collectors / source data setup. 

> Set the Collector name and Source name to be the same as set in section 1.

now

> Set the Collector name and Source name to be the same as set in [Sumo Logic Collectors setup](https://github.com/nuodb/nuodb-helm-charts/tree/agibbons/insights-sumologic/incubator/insights-sumologic#sumo-logic-collectors)